### PR TITLE
alpha development merge

### PR DESCRIPTION
**Pull request by imhta**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                        <li id="iphone"></li>
+                        <li id="ipad"></li>
+                        <li id="itunes"></li>
+                        <li id="support"></li>
+                        <li id="search"><input type="text"></li>
+                    </ul>        
+                </nav>
+                <header>
+                    <div class="heading"></div>
+                    <div class="sub-heading"></div>
```
Comment (Torres-ssf) [inline index.html]: Nice job loading the text images using the div tag. But all of these 3 texts must be a link. Put each div inside an `a` tag. Three `a` and three `div`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                        <li id="ipod"></li>
+                        <li id="iphone"></li>
+                        <li id="ipad"></li>
+                        <li id="itunes"></li>
+                        <li id="support"></li>
+                        <li id="search"><input type="text"></li>
+                    </ul>        
+                </nav>
+                <header>
+                    <div class="heading"></div>
```
Comment (Torres-ssf) [inline index.html]: Put each div inside an a tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                    <div class="sub-heading"></div>
+                    <div class="sm-heading"></div>
```
Comment (Torres-ssf) [inline index.html]: Put each div inside an a tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                <header>
+            </section>
+
+            <div class="sm-img-wrappper">
```
Comment (Torres-ssf) [inline index.html]: In this project, the `footer` must start here. Divide it into two parts. Top and bottom. This is the top.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+                <div class="a-img"></div>
```
Comment (Torres-ssf) [inline index.html]: Make all those images a link. Same approach of the 3 main texts, put each one inside an `a` tag.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+                <div class="b-img"></div>
```
Comment (Torres-ssf) [inline index.html]: Make all those images a link. Same approach of the 3 main texts, put each one inside an `a` tag.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+                <div class="c-img"></div>
```
Comment (Torres-ssf) [inline index.html]: Make all those images a link. Same approach of the 3 main texts, put each one inside an `a` tag.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+                <div class="d-img"></div>
```
Comment (Torres-ssf) [inline index.html]: Make all those images a link. Same approach of the 3 main texts, put each one inside an `a` tag.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
Hi @imhtapm 
Nice work on this project. You got almost everything right, but we need to do some changes before I give this project a 👍. I've reviewed your code and highlighted the place that needs to be fixed. 

- [x] There are some errors in your HTML code. Check them using a validator. You can use this one: https://validator.w3.org/

- [x] The 3 main text from this page must be links. You did a nice job loading the text image using the div's background. Put each one of them inside its own `a` tag to make them links.

- [x] In this project the `footer` must start here at `div class="sm-img-wrappper">`.

- [x] Use the same approach of the 3 main links in the 4 promo images. Put each one of the 4 `div` inside an `a` tag, making them a link.

Kindly make these changes and submit for code review again.

**Comment by maelfosso** [comment]
Nice job,
You're almost done. Just one thing to fix

- [x] As said up, Use the same approach of the 3 main links in the 4 promo images. Put each one of the 4 div inside an a tag, making them a link. We are talking about images inside `<div class="sm-img-wrappper">`

Kindly make these changes and submit for code review again.

**Comment by imhta** [comment]
> Nice job,
> You're almost done. Just one thing to fix
> 
> * [x]  As said up, Use the same approach of the 3 main links in the 4 promo images. Put each one of the 4 div inside an a tag, making them a link. We are talking about images inside `<div class="sm-img-wrappper">`
> 
> Kindly make these changes and submit for code review again.

thank you, I am fixing it. I have a doubt, why I want to do this way? and what is the difference?

**Review by bellom — APPROVED** [review]
Hi @imhtapm 
## Congratulation :+1: 
_Go ahead and merge to master and don't forget to submit project completion form._

**Pull request merged**